content/en/security_platform/application_security/setup_and_configure.md

```diff
@@ -190,9 +190,26 @@ The data that you’re collecting with Datadog can contain sensitive information
 
 The Datadog Agent and some tracing libraries have options available to address these situations and modify or discard spans. See [APM Data Security][1] for details that also apply to Application Security.
 
+## Exclusion filters
+
+There may be a time when an Application Security signal is a false positive. For example, a targeted service is repeatedly generating the same signal, but the signal has been reviewed and is not a threat.
+
+You can set an exclusion filter, which ignore events from a rule, to eliminate these noisy signal patterns and focus on legitimate suspicious requests.
```

Comment:
> There may be a time when an Application Security signal is a false positive. For example, a targeted service is repeatedly generating the same signal, but the signal has been reviewed and is not a threat.

A few language tweaks to consider:
- A service doesn't generate signals. AppSec detects signals based on traffic it receives

> You can set an exclusion filter, which ignore events from a rule, to eliminate these noisy signal patterns and focus on legitimate suspicious requests.
Ignore security events from the event rules